app/assets/stylesheets/config/_spaces.css

```diff
@@ -0,0 +1,64 @@
+/* paddings */
+.p-top-15 {
```

Comment:
**idea:** I tend to prefer using numbers for standardization. So `p-top1`, `p-top2`, etc. That way you think in proportions instead of sizes.

You can see an example in my vars here: https://github.com/tcione/tales.dev/blob/main/sass/_base.scss

Reply:
Using the number 15/30/60 helped me understand a bit faster which one I was using, BUT I totally agree with you. I think on the long run this makes more sense. Thinking in terms of proportions will make it easier to change things if needed. 

I took a look at your file, and decided to go with s, m, and l for our sizes.